css/print.css

```diff
@@ -26,3 +26,59 @@ p {
+    height: 75mm;
+    content: ' ';
+  }
+}
+
+:root {
+  counter-reset: pageNumber;
+  counter-set: pageNumber 1;
+  --page-number-style: lower-roman;
+}
+
+@page {
+  counter-increment: pageNumber 1;
```

Comment:
If you're using `paged.js`, this a) is not necessary and b) [will not work](https://github.com/pagedjs/pagedjs/issues/180).